src/pem_common_plug.c

```diff
@@ -37,6 +37,8 @@ struct fmt_tests pem_tests[] = {
 	{NULL}
 };
 
+static int warned = 0;
```

Comment:
Please move this to inside of `pem_valid`.

Reply:
Ack